src/sentry/search/snuba/backend.py

```diff
@@ -37,14 +36,14 @@ def snuba_str_to_datetime(d):
 
 
 def calculate_priority_cursor(data):
-    times_seen = sum(data['times_seen'])
-    last_seen = max(int(to_timestamp(snuba_str_to_datetime(d)) * 1000) for d in data['last_seen'])
+    times_seen = data['times_seen']
+    last_seen = int(to_timestamp(snuba_str_to_datetime(data['last_seen'])) * 1000)
```

Comment:
What's generally going on here (and directly below) is we no longer deal with lists of values, because we don't have to "merge" hash data client side. We get one value per `group_id` from Snuba and we're done.